.eslintrc.json

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": [
+    "next/core-web-vitals",
+    "@strv/react",
+    "@strv/typescript",
+    "@strv/typescript/react"
+  ],
```

Comment:
Next.js is already started with a `next/core-web-vitals` ESLint config. Here, we are adding:

- [`@strv/react`](https://github.com/strvcom/code-quality-tools/blob/master/packages/eslint-config-react/index.js)
- [`@strv/typescript`](https://github.com/strvcom/code-quality-tools/blob/master/packages/eslint-config-typescript/index.js)
- [`@strv/typescript/react`](https://github.com/strvcom/code-quality-tools/blob/master/packages/eslint-config-typescript/react.js)

Follow the links above to check what this configuration looks like 😉